frontend/src/components/reports/PendingReportsTab.vue

```diff
@@ -5,73 +5,75 @@
         <h2>Report Details</h2>
       </v-col>
       <v-col cols="12" md="7">
-        <FacilityFilter :loading="loading" :defaultShowInput="true" justify="end" @facility-filter-changed="facilityFilterChanged" />
+        <FacilityFilter v-if="!isEmpty(pendingReports)" :loading="loading" :defaultShowInput="true" justify="end" @facility-filter-changed="facilityFilterChanged" />
       </v-col>
     </v-row>
 
-    <v-skeleton-loader :loading="processing" type="table-tbody">
-      <v-data-table :headers="headers" :items="filteredPendingReports" item-key="surveyResponseReferenceNumber" density="compact">
-        <template #[`item.alertType`]="{ item }">
-          <v-icon v-if="item.alertType === 'Due'" color="warning">mdi-alert</v-icon>
-          <v-icon v-else-if="item.alertType === 'Overdue'" color="error">mdi-alert-circle</v-icon>
-          <span class="pl-1">{{ item.alertType }}</span>
+    <v-skeleton-loader :loading="loading" type="table-tbody">
+      <AppAlertBanner v-if="isEmpty(pendingReports)" type="info">
+        <div>You are up to date with your monthly reports.</div>
+      </AppAlertBanner>
+      <v-data-table v-else :headers="headers" :items="filteredPendingReports" item-key="surveyTemplateId" density="compact" class="soft-outline">
+        <template #[`item.alert`]="{ item }">
+          <span v-if="isOverdue(item)">
```

Comment:
Span might be ok here but if you want to have an if-else without creating an extra wrapper you can use template as an invisible wrapper: https://vuejs.org/guide/essentials/conditional.html#v-if-on-template.

Reply:
sorry I'm not sure how it works in my case since I have v-if and v-else to show 2 icons

Reply:
If you use <template v-if=""><template v-else> then it won't create an extra span holding your contents since template is an invisible wrapper. They're already contained in a td anyways. But if you want the span then it's fine.

Reply:
I think I see what you mean. I've learned something new today. thank you!